### ISSUE TITLE
does this work only for frontend?

### ISSUE THREAD

**Issue by LouNik1984** [ericthehacker/magento-themefallbackfix#4]
Hi, it works nice for the frontend, however I noticed that you used the frontend node in your config, so I wonder if this fix can be applied to admin themes also, that would be awsome! It's just a matter of placing the observer in the global node instead of frontend node, or admin need a totally different event identifier? Or cannot it be done at all?

**Comment by ericthehacker** [comment]
@LouNik1984 
Since custom theming of the admin is uncommon, I had really only considered the front end. It _should_ be just as easy as changing the observer to `global` instead of `frontend` as you said. Could you give it a try and see if it works?

**Comment by LouNik1984** [comment]
Yes, of course, I'll try this weekend and I'll let you know what happens
